app/routes/_gcn.circulars._archive._index/route.tsx

```diff
@@ -68,10 +70,27 @@ export async function action({ request }: ActionFunctionArgs) {
   const subject = getFormDataString(data, 'subject')
   if (!body || !subject)
     throw new Response('Body and subject are required', { status: 400 })
-  return await put(
-    { subject, body, submittedHow: 'web' },
-    await getUser(request)
-  )
+  const intent = getFormDataString(data, 'intent')
+  const user = await getUser(request)
+  const circularId = getFormDataString(data, 'circularId')
+  let result
+  if (circularId) {
+    await putVersion(
+      {
+        body,
+        circularId: parseFloat(circularId),
+        subject,
+      },
+      user
+    )
+    result = await get(parseFloat(circularId))
+    if (intent === 'distribute') {
+      // TODO: Send Email
+    }
```

Comment:
What's this for?

Reply:
Woops that was left over from a previous path I was going down. I'll remove it in the next commit